.github/workflows/ci.yaml

```diff
@@ -58,6 +59,14 @@ jobs:
               - crates/ruff_dev/**
               - scripts/*
 
+            code:
+              - crates/*
+              - scripts/*
```

Comment:
```suggestion
              - crates/**
              - scripts/**
```

Reply:
Nice catch, I have copypasted this form existing code.